libs/cart/src/drivers/magento/queries/fragments/cart-item.ts

```diff
@@ -26,6 +27,6 @@ export const cartItemFragment = gql`
       }
     }
   }
-  ${productFragment}
+  ${magentoProductFragment}
```

Comment:
I don't agree with this change. We shouldn't be retrieving the full product on the cart. It's wasteful of bandwidth.

Reply:
It isn't actually any different. The other fragment query in cart was just wrong. We could do another PR to change what we get back if you want. Here is the old one:
```
export const productFragment = gql
  fragment product on MagnetoProduct {
    id
    image {
      label,
      url
    }
    name
    description      <------------this will cause a graphQL failed call. needs to be "description { html }"
    sku
    url_key
    price {
      regularPrice                             <-----------------------This will cause a failure
    }
    media_gallery_entries                             <---------------this will cause a failure
    short_description                                    <---------------this will cause a failure
  }
;
```
So yea there's really no difference. We can trim the request down later?

Reply:
That's fine.